spec/reek/cli/application_spec.rb

```diff
@@ -0,0 +1,10 @@
+require_relative '../../spec_helper'
+require_lib 'reek/cli/application'
+
+RSpec.describe Reek::CLI::Application do
+  describe '#initialize' do
+    it 'works' do
+      Reek::CLI::Application.new []
+    end
```

Comment:
I dont get it. What's the point of this test? Satisfying simplecov? Shouldnt we rather have proper specs?